<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VidyaLaptop" failures="1" tests="1" name="Delete User Tests" time="0.109" errors="0" timestamp="2024-10-18T10:35:17 CDT">
  <testcase classname="testNg.DeleteUserTests" name="givenValidUserId_whenDeleteUser_thenUserIsDeletedSuccessfully" time="0.107">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at testNg.DeleteUserTests.givenValidUserId_whenDeleteUser_thenUserIsDeletedSuccessfully(DeleteUserTests.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- givenValidUserId_whenDeleteUser_thenUserIsDeletedSuccessfully -->
</testsuite> <!-- Delete User Tests -->
